src/Backend/Modules/MediaLibrary/Js/MediaLibraryHelper.js

```diff
@@ -841,15 +1032,9 @@ jsBackend.mediaLibraryHelper.upload =
             validation: {
                 allowedExtensions: jsBackend.data.get('MediaLibrary.mediaAllowedExtensions')
             },
-            scaling: {
-                includeExif: true,
-                sendOriginal: false,
-                sizes: [
-                    {name: "", maxSize: 3000}
```

Comment:
I updated the comment, I can maybe see if I can add something to manually check on that but the way you say to fix it will break the cropper, that's why the comment is there